region/norfair/west/Ice Beam Gate Room.json

```diff
@@ -392,6 +403,12 @@
         {"refill": ["Energy", "Missile", "Super", "PowerBomb"]}
       ]
     },
+    {
+      "link": [3, 3],
+      "name": "Crystal Flash",
+      "requires": ["h_canCrystalFlash"],
```

Comment:
3 is safe only if you spend some ammo then farm it back

Reply:
True, added an enemyKill requirement to this now.